pkgs/top-level/aliases.nix

```diff
@@ -1040,6 +1040,7 @@ mapAliases ({
     plasma-integration
     plasma-nm
     plasma-pa
+    plasma-sdk
```

Comment:
Please do not add aliases for new packages. Aliases are for backwards compatibility.

Reply:
Fixed